### PR TITLE
document stock tracker app

### PR DESCRIPTION
**Pull request by maxdml**
Screenshot 2024-12-09 at 12 12 54

### CODE REVIEW

File: docs/python/examples/stock-tracker.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+displayed_sidebar: examplesSidebar
+sidebar_position: 10
+title: Stock Tracker
+---
+
+In this example, we use DBOS to build and deploy an app that tracks stock prices and send SMS alerts when the price of a stock crosses a certain threshold. We certainly don't want to miss a good opportunity to trade, so it's important this workflow is punctual and reliable!
+
+You can see the application live [here](https://max-stock-prices.cloud.dbos.dev/). All source code is [available on GitHub](https://github.com/dbos-inc/dbos-demo-apps/tree/main/python/stock-prices).
```
Comment (kraftp) [inline docs/python/examples/stock-tracker.md:9]: Do the alerts actually work?  Who are they sent to?

File: docs/python/examples/stock-tracker.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    DBOS.logger.info(f"Stock price for {symbol} is {data['Close'].iloc[0]}")
+    return data['Close'].iloc[0]
+```
+
+## Sending SMS alerts
+
+We'll use the Twilio API to send SMS alerts and write a DBOS [step](../tutorials/step-tutorial) to make it durable. You can sign up for a free Twilio account at https://www.twilio.com/try-twilio. We'll need a few environment variables to store our Twilio account SID, auth token and phone number.
+
+```python
+twilio_account_sid = os.environ.get('TWILIO_ACCOUNT_SID')
```
Comment (kraftp) [inline docs/python/examples/stock-tracker.md:70]: Where do the alerts come from?

File: docs/python/examples/stock-tracker.md (hunk 1 above)
Comment (maxdml) [reply docs/python/examples/stock-tracker.md:9]: The SMS are sent to Twilio. You can see them in your twilio account.

The only issue is that, with a free account, they don't get actually sent to the recipient. If you upgrade to a paying account, then they'll go through.

File: docs/python/examples/stock-tracker.md (hunk 2 above)
Comment (maxdml) [reply docs/python/examples/stock-tracker.md:70]: The DBOS workflow will call the Twilio API. Twilio is responsible for delivering the SMS.

**Review by kraftp — APPROVED** [review]
This needs a tile on the examples page, but otherwise looks good!

**Pull request merged**